### ISSUE TITLE
use websockets to communicate with extension bridge.

### ISSUE THREAD

**Issue by jsoverson** [jsoverson/hackium#11]

**Comment by jbransom** [comment]
Is there a branch of `puppeteer-extensionbridge` that already has the ExtensionBridge class converted to use websocket or would that also need to be enhanced?

**Comment by jsoverson** [comment]
No, nothing has started. This is a tracking task for myself (or any motivated party)
